src/kbmod/search/stamp_creator.cpp

```diff
@@ -190,8 +190,8 @@ std::vector<RawImage> StampCreator::get_coadded_stamps_gpu(ImageStack& stack,
         // should not accept images of different sizes being added to the stack
         // and then get rid of all these for loops in the code
         auto& sci = stack.get_single_image(t).get_science().get_image();
-        assertm(sci.get_width() == width, "Stack image " + t + " has different width than 0th image.");
-        assertm(sci.get_height() == height, "Stack image " + t + " has different width than 0th image.");
+        assertm(sci.cols() == width, "Stack image has different width than 0th image.");
```

Comment:
Above you replaces `cols` and `rows` with `get_width` and `get_height` but now you're doing the opposite. Is there a particular logic you're applying? Asking so I can do the same.

Reply:
`sci` is an Eigen image (from the `get_image()`) and the statement was giving me a compilation error that the eigen image didn't have `get_height()` and `get_width()`.

Above it was the opposite. I was getting a compile error that RawImage did not have `cols()` or `rows()`.

Reply:
These two compile errors were the ones where it was compiling fine for me all morning and then I refreshed something and it started giving me errors on unchanged lines.